internal/tools/aws/database.go

```diff
@@ -66,7 +68,27 @@ func (d *RDSDatabase) Teardown(keepData bool, logger log.FieldLogger) error {
 		return errors.Wrap(err, "unable to delete RDS DB cluster")
 	}
 
-	logger.Debug("AWS RDS DB cluster deleted")
+	logger.Debug("AWS RDS db cluster deleted")
```

Comment:
I think this log message should be either removed or altered so that the stage that is complete here is reported in a way that is more distinct from the log message on line 91